ci-operator/step-registry/ibmcloud/provision/vpc/ibmcloud-provision-vpc-commands.sh

```diff
@@ -99,14 +103,19 @@ function waitAvailable() {
 }
 
 function create_vpc() {
-    local preName="$1" vpcName="$2" resource_group="$3" num_subnets_pair_per_zone="${4:-1}"
+    local preName="$1" vpcName="$2" rg="$3" num_subnets_pair_per_zone="${4:-1}"
     local zone zone_cidr zone_cidr_main subnetName subnet_cidr_main subnets_pair_idx subnets_idx
 
+    echo "Creating vpc $vpcName under $rg ..."
+    export IBMCLOUD_TRACE=true
     # create vpc
-    "${IBMCLOUD_CLI}" is vpc-create ${vpcName} --resource-group-name ${resource_group}
+    "${IBMCLOUD_CLI}" is vpc-create ${vpcName} --resource-group-name "${rg}"
```

Comment:
```suggestion
    IBMCLOUD_TRACE=true "${IBMCLOUD_CLI}" is vpc-create ${vpcName} --resource-group-name "${rg}"
```
So that you do not have to export IBMCLOUD_TRACE=true, and later set it back to false.